nototools/generate_dingbats_html.py

```diff
@@ -864,7 +864,10 @@ def _flagged_name(cp, flag_sets):
   is a map from flag name to a tuple of cp set and boolean.
   True means add flag if cp in set, False means add flag if it is
   not in the set."""
-  name = unicode_data.name(cp)
+  try:
+    name = unicode_data.name(cp)
+  except:
+    raise Exception('no name for %04X' % cp)
```

Comment:
Typos don't.  For example if I specify a unicode range and it includes undefined codepoints by mistake.